README.md

```diff
@@ -45,11 +39,12 @@ To ensure your services stay robust, PSO self-heals – so you’re protected ag
   - Latest iSCSI initiator software for your operating system (Optional, required for iSCSI connectivity)
   - Latest NFS software package for your operating system (Optional, required for NFS connectivity)
   - Latest FC initiator software for your operating system (Optional, required for FC connectivity, *FC Supported on Bare-metal K8s installations only*)
-  - An NTP implementation (such as `ntpd` or `chronyd`) is installed and running on all Kubernetes cluster nodes.
+  - **An NTP implementation (such as `ntpd` or `chronyd`) is installed and running on all Kubernetes cluster nodes**
```

Comment:
[Minor] "An NTP daemon", would that be better?

Reply:
That is actually the official way of describing ntpd, chronyd

Reply:
Sure, works for me.

Reply:
Sadly I am a bit of an NTP nerd - I built and managed a Stratum 0 server back in the UK for one of the main UK Critical Infrastructure providers

Reply:
@sdodsley which one is official? "implementation"? I changed it to daemon but can change it back

Reply:
Official is "implementation", I've taken us back to that.